.github/workflows/nightly-spread.yaml

```
@@ -0,0 +1,70 @@
+name: Nightly spread executions
+
+# See https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onschedule
+on:
+  schedule:
+    - cron: '0 2 * * *'
+    - cron: '0 3 * * *'
+    - cron: '0 6 * * *'
+
```

would this make sense as well?
```suggestion
  workflow_dispatch:
    inputs:
      kind:
        type: string
        description: "Workflow to trigger, one of: nightly, experimental, from-current"
        default: ""
 
```

---

I like it! We can do even better and use choice so people don't have to write out the string